common/views/components/IIIFViewer/IIIFViewer.js

```diff
@@ -0,0 +1,339 @@
+// @flow
+import styled from 'styled-components';
+import { classNames, spacing, font } from '@weco/common/utils/classnames';
+import NextLink from 'next/link';
+import { itemUrl } from '@weco/common/services/catalogue/urls';
+import {
+  type IIIFCanvas,
+  type IIIFImageService,
+} from '@weco/common/model/iiif';
+import Paginator, {
+  type PaginatorRenderFunctionProps,
+  type PropsWithoutRenderFunction as PaginatorPropsWithoutRenderFunction,
+} from '@weco/common/views/components/RenderlessPaginator/RenderlessPaginator';
+import Control from '@weco/common/views/components/Buttons/Control/Control';
+import IIIFResponsiveImage from '@weco/common/views/components/IIIFResponsiveImage/IIIFResponsiveImage';
+
+const IIIFViewerPaginatorButtons = styled.div.attrs(props => ({
+  className: classNames({
+    'flex absolute flex--h-center': true,
+  }),
+}))`
+  left: ${props => (props.isThumbs ? 'auto' : '50%')};
+  right: ${props => props.theme.spacingUnit}px;
+  bottom: ${props => (props.isThumbs ? '50%' : props.theme.spacingUnit + 'px')};
+  transform: ${props =>
+    props.isThumbs ? 'translateY(50%)' : 'translateX(-50%)'};
+
+  ${props =>
+    props.isThumbs &&
```

Comment:
It still feels like we have a completely different set of styles for `isThumbs`- could we potentially compose rather than have loads of conditionals?